dev/tools/gen_defaults/lib/input_decorator_template.dart

```diff
@@ -24,9 +24,9 @@ class _${blockName}DefaultsM3 extends InputDecorationTheme {
   @override
   TextStyle? get hintStyle => MaterialStateTextStyle.resolveWith((Set<MaterialState> states) {
     if (states.contains(MaterialState.disabled)) {
-      return TextStyle(color: Theme.of(context).disabledColor);
+      return TextStyle(color:${componentColor('md.comp.filled-text-field.disabled.supporting-text')});
```

Comment:
nit: might be better to add a space after "color:"

Reply:
good catch! thanks